Startup.cs

```diff
@@ -58,20 +75,53 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAsertoAuthorization(options =>
             {
                 Configuration.GetSection("Aserto").Bind(options);
-                options.ResourceMapper = AuthzResourceContext.Instance.ResourceMapper;
+                options.IdentityMapper = AuthzIdentityContext.Instance.IdentityMapper;
             });
             //end Aserto options handling
 
+            var checkResourceRules = new Dictionary<string, Func<string, HttpRequest, Struct>>();
+
+            checkResourceRules.Add("member", (policyRoot, httpRequest) =>
+            {
+                Struct result = new Struct();
+                if (httpRequest.Method == "POST")
```

Comment:
Why do we need this check? I thought we only use this "member" rule on the POST route?